client/asset/btc/btc.go

```diff
@@ -1464,25 +1485,41 @@ func (btc *baseWallet) legacyBalance() (*asset.Balance, error) {
 }
 
 // feeRate returns the current optimal fee rate in sat / byte using the
-// estimatesmartfee RPC.
-func (btc *baseWallet) feeRate(_ RawRequester, confTarget uint64) (uint64, error) {
-	feeResult, err := btc.node.estimateSmartFee(int64(confTarget), &btcjson.EstimateModeConservative)
+// estimatesmartfee RPC or an external API if configured and enabled.
+func (btc *baseWallet) feeRate(confTarget uint64) (uint64, error) {
+	// Local estimate first. TODO: Allow only external (nil local).
+	feeRate, err := btc.localFeeRate(btc.ctx, btc.node, confTarget) // e.g. rpcFeeRate
+	if err == nil {
```

Comment:
what if we also check `feeRate != 0`? This way if it returns 0 we can also check an external fee rate if the user wants to